integreat_cms/cms/views/imprint/imprint_form_view.py

```diff
@@ -8,13 +8,16 @@
 from django.utils.translation import gettext as _
 from django.views.generic import TemplateView
 
+from integreat_cms.cms.utils.translation_utils import translate_link
+
```

Comment:
```suggestion
```
Could you also use a relative import for this?

Reply:
Sure